libid/complex.cpp

```diff
@@ -0,0 +1,536 @@
+// SPDX-License-Identifier: GPL-3.0-only
+//
+// Complex.cpp: interface for the Complex class.
+//
+//////////////////////////////////////////////////////////////////////
+#include "Complex.h"
+
+//Complex::Complex(void)
+//    {
+//    }
+
+Complex::~Complex(void)
+    {
+    }
+
+Complex Complex::operator=(const double & value)	// Assignment to double Operator
```

Comment:
There's no benefit to accepting a simple value like `double` by const reference; just use `operator=(double value)`

The return type of `operator=` is `Complex &` not `Complex`